www/site/content/docs/chef-workstation/troubleshooting.md

```diff
@@ -12,27 +12,7 @@ Chef Workstation logs are stored in ` ~/.chef-workstation/logs`.
 
 ## Uninstall instructions
 
-### Mac
-
-Run the following code in your terminal:
-
-```bash
-rm -rf /opt/chef-workstation;
-chefdk_binaries="berks chef chef-apply chef-shell chef-solo chef-vault cookstyle dco delivery foodcritic inspec kitchen knife ohai push-apply pushy-client pushy-service-manager chef-client"
-  binaries="chef-run chefx $chefdk_binaries"
-
-for binary in $binaries; do
-  rm -f $PREFIX/bin/$binary
-done
-```
-
-### Windows
-
-Use **Add / Remove Programs** to remove the Chef Workstation product on the Microsoft Windows platform.
-
-### Linux
```

Comment:
Can we just remove this item from troubleshooting entirely ?

Reply:
I don't have a problem with removing it. I just noticed the duplication and it was wrong here.

@kagarmoe @mjingle What say you?

Reply:
That's fine, as long as you provide a redirect, which you did!